### ISSUE TITLE
Items don't load

### ISSUE THREAD

**Issue by bboy11501** [tonesto7/homebridge-smartthings#298]
So I have my setup working, I know this because the smartthings hub shows up in homekit. I can arm it and disarm it etc, and get notifications, but none of the devices attached show up. I have set them to show but they do not. They are lutron switches any ideas?

**Comment by aaronfisher** [comment]
I am having the exact same issue but with zigbee switches. I configure to show up in the smartapp, save, and then nada. My alarm also shows in Home app but no Smartthings accessories.

**Comment by bboy11501** [comment]
> I am having the exact same issue but with zigbee switches. I configure to show up in the smartapp, save, and then nada. My alarm also shows in Home app but no Smartthings accessories.

So I just decided to get the lutron to connect via homekit directly, it was easier but nothing shows 2 ways so I'm still trying to figure that part out.

**Comment by aaronfisher** [comment]
I found a temporary solution. Apparently every time you configure a SmartThings device in the Smartapp, it CHANGES the the app_id and access_token so before you click save, you have to grab them again and update the Homebridge SmartThings plugin on homebridge.local. @bboy11501

**Comment by bboy11501** [comment]
> I found a temporary solution. Apparently every time you configure a SmartThings device in the Smartapp, it CHANGES the the app_id and access_token so before you click save, you have to grab them again and update the Homebridge SmartThings plugin on homebridge.local. @bboy11501

no, I got it to show up in homebridge, but to use lutron and smartthings u need to link the account, but lutron can just connect directly to homekit so I did that. My smartthings bridge shows up in homekit, but my questions is for this to work do the items have to be added directly to smartthings hub or can they just be linked via another service?

**Comment by pdlove** [comment]
@aaronfisher If your App_id and access_token are changing then you're installing the app multiple times. There's a different area Smartthings that shows the existing apps. 
@bboy11501 You can keep Lutron connected straight to HomeKit and use the Smartthings through Homebridge on HomeKit, but the downside is that you can't do the rule activations on smartthings. Like if there is motion then turn on a light. But you can control it all from Home on your phone and set rules in Home that fire between the devices.
I do know that I used to have my lutron hub connected to smartthings and from there in HomeKit. It's been a few years so I can't verify that still works.

**Comment by bboy11501** [comment]
> @aaronfisher If your App_id and access_token are changing then you're installing the app multiple times. There's a different area Smartthings that shows the existing apps.
> @bboy11501 You can keep Lutron connected straight to HomeKit and use the Smartthings through Homebridge on HomeKit, but the downside is that you can't do the rule activations on smartthings. Like if there is motion then turn on a light. But you can control it all from Home on your phone and set rules in Home that fire between the devices.
> I do know that I used to have my lutron hub connected to smartthings and from there in HomeKit. It's been a few years so I can't verify that still works.

right, when it was going via smartthings the lights wouldn't show up in homekit, so I just moved it over will fiddle with it more
